### ISSUE TITLE
Signup Cliff with Uservoice Plan

### ISSUE THREAD

**Issue by sethrubenstein** [PixelRocket-Biz/cliff-michaels#238]

**Comment by cliffmichaelsgithub** [comment]
What's UserVoice?

**Comment by sethrubenstein** [comment]
Thats the helpdesk support widget.

**Comment by cliffmichaelsgithub** [comment]
Lol - what's that? Is it a one time or subscription fee I need to let Cindy know about?

**Comment by sethrubenstein** [comment]
We spoke about this last night. It's the help bubble in the bottom right corner of the screen, right now when people submit a help request it returns a pixelrocket.biz email address. We're signing you up with a new one so that it can come from your address. It'll cost you 12 dollars a month.

**Comment by cliffmichaelsgithub** [comment]
Got it. Just didn't know useVoice was name. I wanted to list all our monthly subscriptions (user names & passwords and customer service phone) as needed, for Cindy's budget records > eg: in cancelling 2 Zoho accounts this weekend > you added Microsoft CRM (monthly?), added mail chimp (monthly?) userVoice (12-15), new stand alone server ($15 ?), All these should be paid by my corp Amex ideally right?

**Comment by cliffmichaelsgithub** [comment]
ANY OTHER FIXED MONTHLY I'm NOT ACCOUNTING FOR?

**Comment by cliffmichaelsgithub** [comment]
Hi Seth & Jen > Can you confirm monthly fixed subscription accounts and the amounts / who we pay / frequency (monthly v annual). Am I missing any other  attached to my Amex Corp card ...?

SETH

  1.  UserVoice (started March 2015). $12/month. Cliff's Corp Amex
  2.  Mail Chimp (started March 2015). $15month
  3.  Microsoft CRM (??)
  4.  New Separate Server (who / how, much, how did we pay)
  5.  VIMEO?

JEN

  1.  MS 365 Exchange for Outlook Sync?
  2.  Dropbox?
  3.  Other

ZOHO CRM & ZOHO CAMPAIGN MONITOR TO BE CANCELLED THIS WEEKEND.


Cliff Michaels & Associates | 310-488-6422
Mortgages | Private Equity | Real Estate
Strategic & Venture Consulting
CliffMichaels.com<http://cliffmichaels.com/funding><http://cliffmichaels.com/funding>/<http://cliffmichaels.com/funding>Funding<http://cliffmichaels.com/funding>
E-Courses | Forums | Retreats
Career & Entrepreneurial Training
CliffMichaels.com<http://cliffmichaels.com> | LinkedIN<linkedin.com/in/cliffmichaels> | Facebook<Facebook.com/cliffmichaels>

**Comment by cliffmichaelsgithub** [comment]
Does this bounce to the customer or to us? i.e: I clicked on the question mark thingy on one of our web pages to do a test ... our site then sent me the attached -- is that what the customer gets too or is that what I got because I'm admin. The message needs to be adjusted for the user so it says from: Support at CliffMichaels.com > email address is support=pixelrocket.biz@uservoice.com (this is an awkard sender to a user - likely to end up in their junk? YES ? NO ?

Where it says "Thanks for the message, we'll get back to you shortly." Can we instead say 

"Thanks for your message to CliffMichaels.com. You're our #1 priority & we will respond shortly. 95% of issues can be solved by reading our FAQ page. Also check your internet connection & make sure you have a compatible browser per our FAQ page. (Link FAQ page).

![screen shot 2015-03-06 at 10 31 13 am](https://cloud.githubusercontent.com/assets/6202539/6531238/7b5b5006-c3ec-11e4-9746-171bb7d74103.png)
